### ISSUE TITLE
Cannot Connect to App Store 

### ISSUE THREAD

**Issue by scr2em** [Cap-go/native-market#4]
does anyone else have this problem ? 

when I use 
```js
 NativeMarket.openStoreListing({
            appId: "APP ID",
 });
 ```
 
 I get 
 
<img width="395" alt="image" src="https://user-images.githubusercontent.com/4671486/201047055-fa676293-e9f9-4ffe-a250-f6bbff21f7ea.png">

*the app id is correct

**Comment by riderx** [comment]
hey yes you are not the first one ! i just found the issue, i did publish a new version who should fix it! `1.10.0`

**Comment by riderx** [comment]
fixed on [v1.0.12](https://github.com/Cap-go/native-market/releases/tag/v1.0.12)

**Comment by scr2em** [comment]
@riderx Thanks, it works now
